app/models/network_port.rb

```diff
@@ -5,6 +5,7 @@ class NetworkPort < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::NetworkManager"
   belongs_to :cloud_tenant
   belongs_to :device, :polymorphic => true
+  belongs_to :load_balancer
```

Comment:
ah, you don't need this one, you can just use :device for it

unless we want to get rid of the polymorphic associations :-)